recipes/imbalanced-learn/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "imbalanced-learn" %}
+{% set version = "0.2.0.dev0" %}
+package:
+  name: imbalanced-learn
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/scikit-learn-contrib/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: ed97184e5665ea71b44222541b4c3d36daf4a8f6cf402df2976538082ff0509d
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.10.4
+    - scipy >=0.17.0
+    - scikit-learn >=0.17.1
```

Comment:
Please drop any dependencies not needed for the build. Probably will only be `python` and `setuptools` left.